src/lib.rs

```diff
@@ -197,20 +268,38 @@ impl KVMi {
         Ok(vcpu_count)
     }
 
-    pub fn get_registers(&self, vcpu: u16) -> Result<(kvm_regs, kvm_sregs, kvm_msrs), Error> {
+    pub fn get_registers(&self, vcpu: u16,) -> Result<(kvm_regs, kvm_sregs,kvm_msr), Error> {
         let mut regs: kvm_regs = unsafe { mem::MaybeUninit::<kvm_regs>::zeroed().assume_init() };
         let mut sregs: kvm_sregs = unsafe { mem::MaybeUninit::<kvm_sregs>::zeroed().assume_init() };
-        let mut msrs: kvm_msrs = unsafe { mem::MaybeUninit::<kvm_msrs>::zeroed().assume_init() };
+        let mut msrs: kvm_msr = unsafe { mem::MaybeUninit::<kvm_msr>::zeroed().assume_init() };
         let mut mode: c_uint = 0;
+        msrs.msrs.nmsrs=6;
+        msrs.entries[0].index = 0x00000174;
```

Comment:
Let's not use hardcoded indexes here.
Please use the x86 crate that I mentionned in Slack

Reply:
It seems x86 crate works only with nightly